src/envs/pybullet_robots.py

```diff
@@ -606,3 +620,53 @@ def inverse_kinematics(
         joint_vals.append(joint_val)
 
     return joint_vals
+
+
+def run_motion_planning(
+        robot: _SingleArmPyBulletRobot, initial_state: JointState,
+        target_state: JointState, collision_bodies: Collection[int], seed: int,
+        physics_client_id: int) -> Optional[Sequence[JointState]]:
+    """Run BiRRT to find a collision-free sequence of joint states.
```

Comment:
the idea is to use this with pybullet_robot2?

Reply:
yeah that's the idea. actually wiring this together is going to be a bit annoying, because we're going to have to have a copy of all of the collision bodies too. if you have any thoughts about the best way to do this lmk...

Reply:
yeah i feel like it's cleanest to just have a second carbon copy of the env that is updated forcibly on each timestep? bc we need to be robust to robot2 doing stupid things during planning and therefore messing up the object states